src/Plugins/RpcServer/ParameterConverter.cs

```diff
@@ -72,6 +72,13 @@ internal static object AsParameter(this JToken token, Type targetType)
             throw new RpcException(RpcError.InvalidParams.WithData($"Unsupported parameter type: {targetType}"));
         }
 
+        internal static T AsParameter<T>(this JToken token)
+        {
+            if (s_conversions.TryGetValue(typeof(T), out var conversion))
+                return (T)conversion(token);
+            throw new RpcException(RpcError.InvalidParams.WithData($"Unsupported parameter type: {typeof(T)}"));
```

Comment:
For value {token} ?

Reply:
I means, mention the received value, otherwise you only will have the type in the error message